### DISCUSSION TITLE
[FEATURE] Etsy Search Analytics

### DISCUSSION THREAD

**Discussion by benjymouse** [etsy/open-api#681] [Ideas]
**Is your feature request related to a problem? Please describe.**
Our customers have expressed a desire to get more insight into search-specific performance of their listings. They want a simple and easy way to probe their search data for good keywords that are getting views and converting into sales. They want to be able to drill down to specific listings, export their own data, and run their shop in a more data-driven manner. 

**Describe the solution you'd like**
An endpoint that provides query, impression, position, visit, conversion rate, revenue, and any other relevant information for sellers' listings. Being able to lookup data by keyword or listing would be highly beneficial.

**Describe alternatives you've considered**
Writing browser extension.

**Comment by kbasali** [comment]
It would be great if this feature was added. Our customers also have a lot of desire in this direction.

**Comment by kbasali** [comment]
any update about these features?

**Comment by philosowaffle** [comment]
ping. 

The Etsy Stats, Ads, and Search Analytics dashboards are completely unusable for anyone trying to do long term analysis, or anyone who wants to combine that data against another external metric (like an advertising run from a 3rd party). Exposing all of the analytics related data via an API would be such an amazing tool for power users and 3rd party apps alike.

**Comment by codyschneiderx** [comment]
Ping.

Any progress on etsy analytics api endpoint?

**Comment by etsyachristensen** [comment]
Nothing new at this time. I'm not sure if we will build this or not. It has not been added to our board.

**Comment by codyschneiderx** [comment]
@etsyachristensen any way I can get momentum for this? Like an upvote system for roadmap? I know there are sellers looking for a better way to optimize their listings and this could be a solution

**Comment by sdumper** [comment]
bump

**Comment by sdumper** [comment]
Is this a work in progress? If so is there a target date?

**Comment by timothymarois** [comment]
Also looking for an update on this.

**Comment by etsyachristensen** [comment]
I've converted this topic into a discussion so that it may be upvoted. I have no details at this time on plans for building this into the system.

**Comment by kbasali** [comment]
Until Etsy adds these features, can we get these details with the crawler?

**Reply by etsyachristensen** [reply]
I'm not sure what you mean by the "crawler".

**Comment by anastasiam** [comment]
Hello! Any updates on this? It would be a very usefull feature. At least an endpoint for getting all the info from the Listing Stats page by date range.
